.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+Proposed changes
```

Comment:
Do we want comments in here to help guide like we have in k3s?

Reply:
I find them a bit noisy, let me try something slightly different.